proto/iotics/api/common.proto

```diff
@@ -165,16 +165,13 @@ message SubscriptionHeaders {
 
 }
 
-// HostID is a unique host identifier.
-message HostID {
-  // Host Identifier string representation
-  string value = 1;
-}
-
-// TwinID is a unique twin identifier.
+// TwinID is the virtual representation of a (physical, purely virtual or hybrid) device,
+// is only ever located in the host it was created.
 message TwinID {
   // Twin Identifier (using DID format)
-  string value = 1;
+  string id = 1;
+  // Host Identifier (using DID format)
+  string hostId = 2;
```

Comment:
Just throwing this into the wild: I liked the differentiation between `id` and `did`. IMHO it helps to communicate to people what they can expect. Are we giving up on the differentiation now?